pyaerocom/io/test/test_read_emep.py

```diff
@@ -69,3 +58,7 @@ def test_preprocess_units():
     except NotImplementedError as e:
         catch_error = True
     assert catch_error == True
+
+if __name__ == '__main__':
+    import sys
+    pytest.main(sys.argv)
```

Comment:
On the long(er) term it could be good to test methods and attributes of ReadEMEP individually. Here is some inspiration, but I am not sure either yet, how to best test classes and class methods:

Example 1 (individual test methods): https://github.com/metno/pyaerocom/blob/v090dev/pyaerocom/io/test/test_read_ebas.py
Example 2 (a test class): https://github.com/metno/pyaerocom/blob/v090dev/pyaerocom/io/test/test_read_ghost.py

But this is not needed now, just something to consider later.